source/includes/_ruby.md

```diff
@@ -42,7 +42,7 @@ bundle install
     <tr>
      	<td><span class="step">2</span></td>
      	<td><p>Download your customized config file, placing it at <code>config/scout_apm.yml</code>.</p>
-        <p class="smaller">Your customized config file is available within your Scout account.</p>
+        <p class="smaller">Your customized config file is available within your Scout account, via [this link](https://apm.scoutapp.com/apps/new_ruby_application_configuration).</p>
```

Comment:
english feedback: `available within your <link>Scout Account</link>` is probably better than `via ...`